test/performance/io/format_vienna_benchmark.cpp

```diff
@@ -0,0 +1,149 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#if __has_include(<seqan/rna_io.h>)
+    #include <seqan/rna_io.h>
+#endif
+
+#include <sstream>
+#include <string>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/io/structure_file/input.hpp>
+#include <seqan3/io/structure_file/output.hpp>
+#include <seqan3/io/structure_file/format_vienna.hpp>
+#include <seqan3/test/performance/units.hpp>
+
+using namespace seqan3;
+using namespace seqan3::test;
+
+inline constexpr size_t iterations_per_run = 1024;
+
+inline std::string const header{"seq foobar blobber"};
+inline std::string const sequence
+{
+    "ACTAGACTAGCTACGATCAGCTACGATCAGCTACGAACTAGACTAGCTACGATACTAGACTAGCTACGATCAGCTACGA"
+    "ACTAGACTAGCTACGATCAGCTACGATCAGCTACGAACTAGACTAGCTACGATACTAGACTAGCTACGATCAGCTACGA"
+    "ACTAGACTAGCTACGATCAGCTACGATCAGCTACGAACTAGACTAGCTACGATACTAGACTAGCTACGATCAGCTACGA"
+    "ACTAGACTAGCTACGATCAGCTACGATCAGCTACGAACTAGACTAGCTACGATACTAGACTAGCTACGATCAGCTACGA"
+    "ACTAGACTAGCTACGATCAGCTACGATCAGCTACGAACTAGACTAGCTACGATACTAGACTAGCTACGATCAGCTACGA"
+    "ACTAGACTAGCTACGATCAGCTACGATCAGCTACGAACTAGACTAGCTACGATACTAGACTAGCTACGATCAGCTACGA"
+};
+inline std::string const structure
+{
+    "(((((((..((((........)))).((((.........)))).....(((((.......))))))))))))......."
+    "(((((((..((((........)))).((((.........)))).....(((((.......))))))))))))......."
+    "(((((((..((((........)))).((((.........)))).....(((((.......))))))))))))......."
+    "(((((((..((((........)))).((((.........)))).....(((((.......))))))))))))......."
+    "(((((((..((((........)))).((((.........)))).....(((((.......))))))))))))......."
+    "(((((((..((((........)))).((((.........)))).....(((((.......))))))))))))......."
+};
+
+inline std::string const vienna_file = []()
+{
+    std::string file{};
+    for (size_t idx = 0; idx < iterations_per_run; idx++)
+        file += "> " + header + "\n" + sequence + "\n" + structure + "\n";
+    return file;
+}();
+
+void write3(benchmark::State & state)
```

Comment:
Can we call it `write_seqan3`? write3 is not very expressive (I know you probably copied it from another benchmark)

Reply:
True, good idea!